src/qgraphicsmozview_p.cpp

```diff
@@ -540,7 +541,12 @@ void QGraphicsMozViewPrivate::OnLocationChanged(const char* aLocation, bool aCan
     if (mLocation != aLocation) {
         mLocation = QString(aLocation);
         mViewIface->urlChanged();
```

Comment:
what is the point to push two similar events such as urlChanged and locationChanged? why not use urlChanged here instead?